README.md

```diff
@@ -443,17 +443,17 @@ For **engineers** in Silicon Valley, the highest—thus not typical—equity lev
-- If you’re coming from an established company to a startup, you may be asked to take a salary cut. This is reasonable, but it’s wise to discuss explicitly how much it is, and when that will be changed up front. For example, you might take 25% below your previous salary, but there can be an agreement that this will be corrected if your performance is strong and the company gets funding.
-- 🔹 Always negotiate non-compensation aspects before agreeing to an offer. If you want a specific role, title, opportunity, visa sponsorship, special treatment (like working from home), or have timing constraints about when you can join, negotiate these early, not late in the process.
+- If you’re coming from an established company to a startup, you may be asked to take a salary cut. This is reasonable, but it’s wise to discuss explicitly how much the cut is, and when your salary will be renegotiated. For example, you might take 25% below your previous salary, but there can be an agreement that this will be corrected if your performance is strong and the company gets funding.
+- 🔹 Always negotiate non-compensation aspects before agreeing to an offer. If you want a specific role, title, opportunity, visa sponsorship, parental leave, special treatment (like working from home), or have timing constraints about when you can join, negotiate these early, not late in the process.
 - ❗ Get all agreements in writing, if they are not in your offer letter.
 - 🔹 If you’re going to be a very early employee, consider asking for a restricted stock grant instead of stock options, and a cash bonus equal to the tax on those options. This costs the company a little extra paperwork (and legal costs), but means you won’t have to pay to exercise. Then, if you file an 83(b) election, you’re simplifying your situation, eliminating the AMT issues of ISOs, and maximizing your chances of qualifying for long-term capital gains tax.
 - Getting multiple offers is always in your interest. If you have competing offers, sharing the competing offers with the company you want to work for can be helpful, granted your offers are competitive.
-  - However, dragging out negotiations excessively so you can “shop around” an offer to other companies is considered bad form by some people, so it’s thoughtful to be judicious and timely to the extent that it’s possible.
-- **Reneging on offers**: Do not accept an offer verbally or in writing unless you’re ready to stand by your word. In practice, people do occasionally accept an offer and then go back on it, or renege. In the United States, this is considered a very bad thing to do, especially if it put the company in a difficult position (they may have declined another key candidate based on your acceptance), and may hurt your reputation in unexpected ways later.
```

Comment:
@RacheltheEditor not sure on this change. What needs to be conveyed is (1) at least for senior roles in the US, this is pretty strongly frowned upon but (2) how bad a faux pas this really is definitely cultural and varies a lot around the world (in some places it's not so bad, and many readers may be new to the US).

Reply:
I get that it's cultural. It's not the reminder that this Guide pertains to U.S. companies I mind, it's the "very bad thing to do" line. 

"While this practice is acceptable in some business cultures around the world, reneging on an offer at a U.S. company can be dangerous for your employers *and* for your reputation. This is especially true for those hired in senior positions, but it's not good form for anyone. Make sure you're ready to take the job when you say yes."